### PR TITLE
Bundle Size reduction - updated poppins font

### PR DESCRIPTION
**Pull request by anikesh-butler**
The bundle size is now ~1Mb.

### CODE REVIEW

File: src/styles/poppins.css [hunk 1]
```diff
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap');
```
Comment (butler-matt) [inline src/styles/poppins.css:1]: Are we sure this is the right approach? This just means the user is going to need to load this library on page load, as opposed to having it included in our bundle.

Did we explore other options here or is this then only approach?

Comment (anikesh-butler) [reply src/styles/poppins.css:1]: Another way is to embed the font files within the bundled JS as before but with the required files only. That'd make the JS bundle again big as before.

Comment (anikesh-butler) [reply src/styles/poppins.css:1]: Initially, font files were not in the bundle files but within the dist directory alongside bundle JS which makes a similar case as here ^ loading the files on page load. Also, the problem with this approach is that the end-user webpack might not be configured enough to handle font files which was the case before and we were not able to see the typeface applied to the end user that's why we embedded every font file into bundle JS which is making the bundle size huge.

Comment (anikesh-butler) [reply src/styles/poppins.css:1]: I believe this is the best way we can apply here to reduce the bundle size, another way is to embed font files again into bundle js and make it at least smaller than before.

**Review by butler-matt — APPROVED** [review]

**Pull request merged**